packages/ui-library/src/components/input-field-text/index.stories.ts

```diff
@@ -111,27 +111,24 @@ export default {
       },
       if: { arg: 'hasLabel', eq: true },
     },
-    // todo it has to be renamed? Checke this
-    showInputIcon: {
+
```

Comment:
Please maintain the consistency with the new-line. Right now they are little bit mixed up.

Reply:
This is ready for approval / merging once this is resolved